types/hapi-pino/index.d.ts

```diff
@@ -22,22 +23,30 @@ declare module '@hapi/hapi' {
 }
 
 declare namespace HapiPino {
-    type LogLevels = 'trace' | 'debug' | 'info' | 'warn' | 'error';
+    type Level = 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace';
+    type SerializerFn = (value: any) => any;
 
     interface Options {
         logPayload?: boolean;
         logRouteTags?: boolean;
         stream?: NodeJS.WriteStream;
-        prettyPrint?: boolean;
-        levelTags?: { [key in LogLevels]: string };
-        allTags?: LogLevels;
-        serializers?: { [key: string]: (param: any) => void};
+        prettyPrint?: boolean | pino.PrettyOptions;
+        tags?: { [key in Level]?: string };
+        allTags?: Level;
+        serializers?: { [key: string]: SerializerFn };
```

Comment:
```suggestion
        serializers?: { [key: string]: pino.SerializerFn };
```

Reply:
oh nice catch - so `pino.SerializerFn` is actually exported, so therefore `pino.Level` is also.